web/views/view_helper.ex

```diff
@@ -0,0 +1,11 @@
+defmodule Pairmotron.ViewHelpers do
+  @doc """
+  Given a user, returns true if that user has a role and that role's
+  is_admin property is true. Errors if the user's role is not loaded
+  and the user has a role. This is to prevent calling the database in
+  a view.
+  """
+  def is_admin?(user) do
+    user.role_id && user.role.is_admin
+  end
```

Comment:
So like the comment says, this errors if the user has a role and that role association is not preloaded. That's okay though, as it should always be preloaded at this point, and if it isn't, it should hard error rather than try to grab the role from the database (like the controller_helper version of this method does).